### DISCUSSION TITLE
Curvature node changes value depending on zoom.

### DISCUSSION THREAD

**Discussion by DaylightIsJustABotanist** [bnpr/Malt#514] [Q&A]
The curvature node value increases with an increased distance between the model and the view. I suspect that this is because of changing screen resolution, but I'm not sure of it. The only thing I have to go off of is that a greater or less than node set to .5 produces the same results regardless of zoom distance. Is there a way to get it to change based on the screen resolution so that the output values remain consistent?

**Comment by pragma37** [comment]
>  Is there a way to get it to change based on the screen resolution so that the output values remain consistent?

Not really, that's the way it is. It's a screen space filter that works by checking contiguous pixels.
You can divide by pixel world size to get a somewhat more consistent result, but it's far from perfect.

![imagen](https://github.com/bnpr/Malt/assets/36610999/34badd03-2e1e-4482-bc0b-cf30e8678b31)

What you are asking for would work closer to ambient occlusion under the hood, but it's not something you can achieve by tweaking the curvature node output.

**Reply by DaylightIsJustABotanist** [reply]
This does actually help a lot more than you'd think, thank you.
